templates/home/index.json-ld

```diff
@@ -41,6 +41,18 @@
 				"language": "fr",
 				"path": "api-fr.html"
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility (Only new SC WCAG 2.1/2.2 Level AA)-English report",
+				"language": "en",
+				"path": "reports/pre-a11y-1-en.html"
+			},
+			{
+				"title": "Accessibility (Only new SC WCAG 2.1/2.2 Level AA)-French report",
```

Comment:
The title should match the actual page title for those report.

Reply:
fixed

Reply:
https://github.com/AtlakM/GCWeb/blob/wet-386-pre-assessment-gc-home-page/templates/home/index.json-ld